pallets/phala/src/compute/base_pool.rs

```diff
@@ -826,9 +826,43 @@ pub mod pallet {
 				.is_ok(),
 				Error::<T>::BurnNftFailed,
 			);
+			Self::remove_properties(cid, nft_id);
 			Ok(())
 		}
 
+		fn remove_properties(cid: CollectionId, nft_id: NftId) {
+			let key: BoundedVec<u8, <T as pallet_uniques::Config>::KeyLimit> = "name"
```

Comment:
if these are constant, can it be a real constant to avoid these `.as_bytes().to_vec()..try_into()` then expect?

Reply:
it could，we will reform it at a no-emergency upgrade.

Reply:
fine, but we need test this, these codes have possible error, it will block the extrinsic call which will be bad

Reply:
it won‘t, at least it won't get panic, and we don't check the Result, so at least it won't be worse, during the motion we wlll check it.